Casks/p/plex.rb

```diff
@@ -9,9 +9,12 @@
 
   livecheck do
     url "https://plex.tv/api/downloads/6.json"
-    regex(/"version"\s*:\s*"(\d(?:\.\d+)*)-([a-f0-9]{8})"/i)
-    strategy :page_match do |page, regex|
-      page.scan(regex).map { |match| "#{match[0]},#{match[1]}" }
+    regex(/(\d(?:\.\d+)*)-([a-f0-9]{8})/i)
+    strategy :json do |json, regex|
+      match = json.dig("computer", "MacOS", "version")&.match(regex)
+      next if match.blank?
+
+      "#{match[1]},#{match[2]}"
```

Comment:
```suggestion
    strategy :json do |json|
      json.dig("computer", "MacOS", "version")&.tr("-", ",")
```

Similar to the `keybase` PR, substituting `-` with `,` would allow us to avoid an explicit regex and simplify the `strategy` block.

It may be worth mentioning that we tend to use `\h` when matching hexadecimal text instead of an explicit `[a-f0-9]`. `\h` isn't portable (iirc, it's specific to Ruby's regex engine) but that's not a concern for us (yet). If we ever get a point where we need portability, we could easily replace instances of `\h` with `[[:xdigit:]]` but `\h` is more terse for the time being.

Reply:
No worries, I'm always glad to help and I appreciate the work you've put into these 👍